cabal.project

```diff
@@ -0,0 +1 @@
+packages: ../xcb-types/xcb-types.cabal ./xcffib.cabal
```

Comment:
I think we don't necessarily want to include this either, as it assumes a local copy of xcb-types.